src/commands/init_mission.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::{Result, bail};
+
+use crate::{config::E4EDMConfig, dataset::build_mission_metadata};
+
+use super::InitMissionArgs;
+
+pub(crate) fn init_mission(args: &InitMissionArgs, config: &mut E4EDMConfig) -> Result<()> {
+    if config.active_dataset.is_none() {
+        bail!("Dataset not active!");
+    }
+    let active_dataset_name = config.active_dataset.clone().unwrap();
```

Comment:
This can be reduced to the following: 
`let active_dataset_name = config.active_dataset.clone().ok_or(anyhow!("Dataset not active!"))?;`

Reply:
I personally feel like this suggestion makes the code less readable than the original. I've implemented the change regardless (since it does also improve repetitiveness), but lmk what you think

Reply:
That's a reasonable opinion, though I personally don't like using `unwrap()` as it introduces the possibility of panicking. 

A good middle ground might be the `let else` statement, as described here: https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html#let-else-statements